features/305-microservice/microservice.md

```diff
@@ -61,6 +82,20 @@ When the user run `skycli app deploy`, the following steps are taken:
 
 When the built image is run, it must expose an HTTP service listening at `port`.
 
+### Environment variable and secrets
+
+```yaml
+deployments:
+  my-awesome-service:
+    type: http-service
+    ....
+    environment:
+    - name: "DEMO_ENV"
+      value: "value of environment"
+      name: "MONGODB_URL"
+      secret: "<secret_name>"
```

Comment:
An idea, would it be better if have a shorthand when user want to use a secret name directly as an env var name too? Just to save a few keystrokes, like:

`secret2env: "<secret_name>"` or a `secret` without a `name`

is identical to

```
name: "<secret_name>"
secret: "<secret_name>"
```

Reply:
`secret` without a `name` is unambiguous.

- `ENV_NAME: "plain value"`
- `ENV_NAME: "<secret_name>"`

is too ambiguous

Reply:
I think we can do `secret` without a `name`, but I will still prefer user provide the `name`  maybe in the guide so user will know they can specify the env name in `skygear.yaml`.